packages/portal/src/components/item/ItemMediaThumbnail.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <div
+    class="item-media-thumbnail text-lowercase"
+    data-qa="item media thumbnail"
+  >
```

Comment:
This div should be a button element as clicking it causes an interaction. Buttons are also keyboard accessible. 
Currently the click is handled by the parent. Consider refactoring the div into a button (`b-button`) and perhaps moving the click event down to this button.

Reply:
Better even than my previous comment, refactor the thumbnail into a link to the specific item media page.